cve_bin_tool/checkers/firefox.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for firefox
+
+https://www.cvedetails.com/product/3264/Mozilla-Firefox.html?vendor_id=452
+
+"""
+from __future__ import annotations
+
+from cve_bin_tool.checkers import Checker
+
+
+class FirefoxChecker(Checker):
+    CONTAINS_PATTERNS: list[str] = []
+    FILENAME_PATTERNS: list[str] = [r"firefox"]
+    VERSION_PATTERNS = [
+        r"ENCODER=Mozilla([0-9]+.[0-9]+(.[0-9]+)?).*Firefox",
```

Comment:
```suggestion
        r"ENCODER=Mozilla([0-9]+.[0-9]+(.[0-9]+)?)",
```
The string provided by helper script was different from when I tried to grep it. And it seems like there was no word `Firefox` in there. I'll file a bug about it.